test/JDBC/expected/14_6__preparation__ISC-sequences-vu-prepare.out

```diff
@@ -11,6 +11,15 @@ go
 Use isc_sequences_dbone;
 go
 
+-- BABEL-2571
```

Comment:
We need a test case for the following scenario
1. user created a guest schema in 14.6/15.1 or lower
2. upgrade to 14.7 or 15.2.
3. Check all behaviors work as expected.

So, I think it's worth to have a separate test file.

Reply:
reference: `datediff_internal_date-before-14_7-or-15_2` in https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1108/files